### PR TITLE
feat: input money

### PR DESCRIPTION
**Pull request by Kaoff**
# Description

Adding Input Money, an input in which you can trim decimals.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refacto 
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentantion 

# How Has This Been Tested?

Storybook and Local project

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/Inputs/CorInputMoney/CorInputMoney.module.scss [hunk 1]
```diff
@@ -0,0 +1,78 @@
+@import '../Input.module';
+
+.cor_input_money {
+    @extend .cor_input;
+
+    &--no-btn {
+        input::-webkit-inner-spin-button,
+        input::-webkit-outer-spin-button {
```
Comment (josermarinr) [inline src/components/Inputs/CorInputMoney/CorInputMoney.module.scss:8]: https://developer.mozilla.org/fr/docs/Web/CSS/::-webkit-inner-spin-button

File: src/components/Inputs/CorInputMoney/CorInputMoney.module.scss [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+        .cor_currency_symbol {
+            align-items: center;
+            color: $grey-7;
+            display: flex;
+            line-height: 1;
+            padding: 0 16px;
+            position: absolute;
+            top: 0;
+            z-index: 2;
```
Comment (josermarinr) [inline src/components/Inputs/CorInputMoney/CorInputMoney.module.scss:51]: refacto ( same line after 63)

**Review by josermarinr — DISMISSED** [review]
some refacto in css, good job

File: src/components/Inputs/CorInputMoney/CorInputMoney.module.scss (hunk 2 above)
Comment (Kaoff) [reply src/components/Inputs/CorInputMoney/CorInputMoney.module.scss]: Done

**Pull request merged**